package.json

```diff
@@ -5,8 +5,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc",
-    "test": "npm run build && mocha --reporter spec",
+    "clean": "rimraf dist",
+    "benchmark": "npm run build && node dist/test/benchmark.js",
+    "build": "npm run clean && tsc",
+    "test": "mocha --require ts-node/register test/**/*.spec.ts --reporter list",
```

Comment:
Changed the reporter to `list` as `spec` took up too much space imo

Reply:
cool